service/src/test/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/AddressMapperTest.java

```diff
@@ -94,4 +97,12 @@ public void shouldMapAddress() {
         assertThat(address.getDistrict()).isEqualTo(DISTRICT);
         assertThat(address.getPeriod()).isEqualTo(period);
     }
+
+    @Test
+    public void shouldMapAddressType() {
```

Comment:
I think the new assertion from this test can be just added to `shouldMapAddress` test (and when statement to `setup`).